web-ui/src/components/guild-results/EditGuildModal.jsx

```diff
@@ -187,6 +187,9 @@ const EditGuildModal = ({ guild = {}, open, onSave, onClose, headerText }) => {
               ? editedGuild.guildMembers.filter(guildMember => guildMember.lead)
               : []
           }
+          isOptionEqualToValue={(option, value) =>
```

Comment:
This prop was missing and I think it is required to compare options to the selected value, just like it is used in the next `AutoComplete`.